powershap/shap_wrappers/shap_explainer.py

```diff
@@ -152,44 +152,58 @@ def explain(
                         np.arange(len(X)), test_size=val_size, random_state=i, stratify=stratify
                     )
 
-            X_train = X.iloc[np.sort(train_idx)]
-            X_val = X.iloc[np.sort(val_idx)]
+            X_train = X.iloc[np.sort(train_idx)].to_numpy()
+            X_val = X.iloc[np.sort(val_idx)].to_numpy()
```

Comment:
Yes, this is also possible. Based on my other comment, I don't think it's necessary, since I'm pretty sure this function is already working on a copy of the data. What do you think?